iree/compiler/Dialect/Flow/Transforms/HLOToHLOPreprocessing.cpp

```diff
@@ -225,6 +225,20 @@ class Lower1x1ConvolutionToDotOp : public OpRewritePattern<mhlo::ConvOp> {
       if (filterShape[dim.getZExtValue()] != 1) return failure();
     }
 
+    // Check dilation & strides are ones.
+    if (op.window_strides()) {
+      auto window_strides = op.window_strides().getValue();
```

Comment:
how about put this to the for loop directly, so we don't need to capture it?
Ie

```
for (auto stride : op.window_strides()...)
```